pull_request_template.md

```diff
@@ -3,6 +3,7 @@ _Describe the contents of the PR, how to manually test it, and provide examples/
 - [ ] I have manually tested the code
 - [ ] The primary ticket number is referenced in PR and commits
 - [ ] I have performed a self-review of the code
-- [ ] I have considered adding automated tests and added or deliberately did not
-- [ ] I have updated the ticket status
-- [ ] I have added feature flags wherever applicable
\ No newline at end of file
+- [ ] I have adding automated tests, or intentionally chose not to
+- [ ] I have updated the ticket state, if applicable
+- [ ] I have updated the ticket deployment info, if applicable
+- [ ] I have added feature flags, wherever applicable
```

Comment:
While we're at it, can we have the self-review item as the last in the list, and the ticket state changes right before that?

Reply:
I mean, ideally I'd like to see them randomized each time, so that people don't get complacent...